### PR TITLE
extract price into his own block from Event Details

### PR DESCRIPTION
**Pull request by mitogh**
Ticket: https://central.tri.be/issues/102653

### CODE REVIEW

File: src/modules/blocks/event-details/block.js [hunk 1]
```diff
@@ -168,22 +168,6 @@ class EventDetails extends Component {
 						/>
 					</div>
 
-					<div className='tribe-editor__event-cost'>
```
Comment (bordoni) [inline src/modules/blocks/event-details/block.js]: Sorry, I might not have made that clear, we want to keep Details block as is.
We are going to keep it for backwards compatibility.

File: src/modules/blocks/event-details/block.js [hunk 2]
```diff
@@ -230,19 +214,6 @@ class EventDetails extends Component {
 							onChange={ ( value ) => setAttributes( { allDay: value } ) }
 						/>
 					</PanelBody>
-					<PanelBody title={ __( 'Price Settings', 'events-gutenberg' ) }>
```
Comment (bordoni) [inline src/modules/blocks/event-details/block.js]: Sorry, I might not have made that clear, we want to keep Details block as is.
We are going to keep it for backwards compatibility.

File: src/modules/blocks/event-details/index.js [hunk 3]
```diff
@@ -59,21 +59,6 @@ export default {
 			source: 'meta',
 			meta: '_EventURL',
 		},
-		eventCost: {
-			type: 'string',
-			source: 'meta',
-			meta: '_EventCost',
-		},
-		eventCurrencySymbol: {
```
Comment (bordoni) [inline src/modules/blocks/event-details/index.js]: We still want this to be part of the classic block.

File: src/views/blocks/event-details.php [hunk 4]
```diff
@@ -37,14 +37,13 @@
...
-<div class="tribe-events-single-section tribe-events-event-meta primary tribe-clearfix">
-	<?php do_action( 'tribe_events_single_event_meta_primary_section_start' ); ?>
+<div class="tribe-events-single-section tribe-events-event-cost tribe-clearfix">
+	<?php do_action( 'tribe_events_single_event_cost_section_start' ); ?>
 
 	<?php $this->template( 'blocks/parts/details' ); ?>
 	<?php $this->template( 'blocks/parts/organizer' ); ?>
 
-	<?php do_action( 'tribe_events_single_event_meta_primary_section_end' ); ?>
+	<?php do_action( 'tribe_events_single_event_cost_section_end' ); ?>
```
Comment (bordoni) [inline src/views/blocks/event-details.php]: Why are we changing this.

File: src/views/blocks/event-details.php [hunk 5]
```diff
@@ -37,14 +37,13 @@
  */
 $time_title = apply_filters( 'tribe_events_single_event_time_title', __( 'Time:', 'events-gutenberg' ), $event_id );
 
-$cost = tribe_get_formatted_cost();
 $website = tribe_get_event_website_link();
 ?>
-<div class="tribe-events-single-section tribe-events-event-meta primary tribe-clearfix">
-	<?php do_action( 'tribe_events_single_event_meta_primary_section_start' ); ?>
+<div class="tribe-events-single-section tribe-events-event-cost tribe-clearfix">
```
Comment (bordoni) [inline src/views/blocks/event-details.php]: We want to keep the Classic Event details.

File: src/views/blocks/event-price.php [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<?php
+$event_id = $this->get( 'post_id' );
+
+$cost = tribe_get_formatted_cost();
+?>
+<div class="tribe-events-single-section tribe-events-event-meta primary tribe-clearfix">
+	<?php do_action( 'tribe_events_single_event_meta_primary_section_start' ); ?>
+    <?php
```
Comment (bordoni) [inline src/views/blocks/event-price.php]: Weird indentation.

File: src/views/blocks/event-price.php [hunk 7]
```diff
@@ -0,0 +1,16 @@
+<?php
+$event_id = $this->get( 'post_id' );
+
+$cost = tribe_get_formatted_cost();
+?>
+<div class="tribe-events-single-section tribe-events-event-meta primary tribe-clearfix">
+	<?php do_action( 'tribe_events_single_event_meta_primary_section_start' ); ?>
```
Comment (bordoni) [inline src/views/blocks/event-price.php]: Lets avoid these filters there, `Tribe__Template` has enough filters for most changes needed.

File: src/views/blocks/parts/details.php [hunk 8]
```diff
@@ -113,14 +112,6 @@
 
 		<?php endif ?>
 
-		<?php
-		// Event Cost
-		if ( ! empty( $cost ) ) : ?>
-
-			<dt> <?php esc_html_e( 'Cost:', 'events-gutenberg' ) ?> </dt>
```
Comment (bordoni) [inline src/views/blocks/parts/details.php]: Classic event Details should stay the same.

File: src/views/blocks/event-details.php (hunk 4 above)
Comment (mitogh) [reply src/views/blocks/event-details.php]: Wrong place :)

**Comment by mitogh** [comment]
Ready for another check @bordoni

**Pull request merged**